CalendarFXView/src/main/java/com/calendarfx/view/popover/EntryPopOverContentPane.java

```diff
@@ -74,6 +74,12 @@ public EntryPopOverContentPane(PopOver popOver, DateControl dateControl, Entry<?
 
         setExpandedPane(detailsPane);
 
+        // Disable auto-hide when user types in time fields, re-enable on focus lost
+        EntryDetailsView entryDetails = (EntryDetailsView) details;
+        entryDetails.getStartTimeField().setOnKeyPressed(e -> popOver.setAutoHide(false));
+        entryDetails.getEndTimeField().setOnKeyPressed(e -> popOver.setAutoHide(false));
+        focusedProperty().addListener((obs, oldVal, newVal) -> { if (!newVal) popOver.setAutoHide(true); });
```

Comment:
This didn't work, I think it may be in MonthEntryViewSkin.java around the entry.intervalProperty().addListener(weakUpdateViewListener); part?